libs/repositories/src/UsdRepository.ts

```diff
@@ -0,0 +1,16 @@
+import { get } from 'http';
+
+export interface UsdRepository {
```

Comment:
Not sure how generic we want to go, and if we'll ever get more currencies, but this could also be modelled like:
CurrencyRepository or FiatRepository which then contains UsdRepository, EurRepository and so on.

Or it's a single CurrencyRepository or FiatRepository in which the actual currency is one of the parameters.

Reply:
I though about this, and because we don't plan to change this in the near feature, I think is fine. If it changes scope, its easy to increase the scope of the repository later.